app/posts/publish-teacher-training-courses/2020-01-23-notifications-mvp.md

```diff
@@ -85,10 +85,112 @@ The Becoming a Teacher team
 
 {% from "email/macro.njk" import appEmail %}
 {{ appEmail({
-  subject: "Course [Course name] (code) has been created",
+  subject: "Course [course_name] (course_code) has been published",
   content: template2
 }) }}
 
+## Email template: A course has been created
+
+{% set template6 %}
+Dear colleague,
+
+A new course has been created in Publish teacher training courses, and your organisation has been added as the accredited body for it.
+
+((Provider_name)) created ((course_name)) ((course_code)) at ((create_course_datetime)). The course has not yet been published on Find postgraduate teacher training. You’ll receive a notification if it is published.
+
+View the course at ((publish_url)).
+
+If you have any questions about this, please notify your contact at ((provider_name)). 
+
+To stop receiving these notifications, or to update your settings, contact becomingateacher@digital.education.gov.uk, or reply to this email.
+
+Regards,
+The Becoming a Teacher team
+{% endset %}
+
+{% from "email/macro.njk" import appEmail %}
+{{ appEmail({
+  subject: "Course [course_name] (course_code) has been created",
+  content: template6
+}) }}
+
+## Email template: A course has been withdrawn
+
+{% set template3 %}
+Dear colleague,
+
+A course, which your organisation is the accredited body for, has been withdrawn from Find postgraduate teacher training.
+
+((provider_name)) withdrew ((course name)) (((course code))) in Publish teacher training courses, at ((attribute_change_datetime)).
+
+This course is no longer visible on Find and candidates can’t apply to it. It also can’t be republished or reopened to applicants in the current cycle.
+
+You can view the withdrawn course in Publish at ((course url)).
+
+If you have any questions about this change, please notify your contact at ((provider_name)). 
+
+To stop receiving these notifications, or to update your settings, contact becomingateacher@digital.education.gov.uk, or reply to this email.
+
+Regards,
+The Becoming a Teacher team
+{% endset %}
+
+{% from "email/macro.njk" import appEmail %}
+{{ appEmail({
+  subject: "Course [course_name] (course_code) has been withdrawn",
+  content: template3
+}) }}
+
+## Email template: Your organisation has been added as the accredited body
+
+{% set template4 %}
+Dear colleague,
+
+Your organisation has been added as the accredited body for a course in Find postgraduate teacher training.
+
+The accredited body for ((course_name)) (((course_code))) was previously ((old accredited body)). This was changed in Publish teacher training courses by ((provider_name)) at ((attribute_change_datetime)).
+
+View the course at ((course_url)).
+
+If you have any questions about this, please notify your contact at ((provider_name)).  
+
+To stop receiving these notifications, or to update your settings, contact becomingateacher@digital.education.gov.uk, or reply to this email.
```

Comment:
Can't they update their settings on Publish?

Reply:
Yes, the templates are actually being updated now to refelct this. I added the originally designed templates.